gltf/gltfpack.cpp

```diff
@@ -488,6 +498,11 @@ static void process(cgltf_data* data, const char* input_path, const char* output
 				append(json_meshes, ",\"indices\":");
 				append(json_meshes, index_accr);
 			}
+			if (settings.keep_extras && !prim.extras.empty())
```

Comment:
Why is this necessary? AFAICT the only "extras" node in the .VRM file from the issue is on the mesh, not on primitive, and that contains targetNames which should already be preserved by gltfpack.